CONTRIBUTING.md

```diff
@@ -14,7 +14,7 @@ The following is a set of guidelines for contributing to the NGINX Ingress Contr
   * [Git Style Guide](#git-style-guide)
   * [Go Style Guide](#go-style-guide)
 
-[Code of Conduct](#code-of-conduct)
+[Code of Conduct](https://github.com/nginxinc/kubernetes-ingress/blob/master/CODE_OF_CONDUCT.md)
```

Comment:
I think we should use relative links here just in case

Reply:
I'd also prefer to use relative, however this is inconsistent with the rest of the style of the contributing guide. 

I'll wait for feedback from @pleshakov on this.

Reply:
I think you are right with consistency. It feels more natural to me to keep them relative (specially with branch or folder changes) but feel free to merge this and maybe we can create another PR to make everything relative.